src/nft/components/details/detailsV2/DataPageTraits.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { TEST_NFT_ASSET } from 'test-utils/constants'
+import { render } from 'test-utils/render'
+
+import { DataPageTraits } from './DataPageTraits'
+
+it('data page trait component does not load with asset with no traits', () => {
+  const { asFragment } = render(<DataPageTraits asset={TEST_NFT_ASSET} />)
+  expect(asFragment()).toMatchSnapshot()
+})
+
+// TODO: add test for trait component with asset with traits when rarity is not randomly generated
+// while rarities are randomly generated, snapshots will never match
```

Comment:
nit: what I usually do is I create a JIRA ticket for each todo and write `TODO(WEB-XXX)` instead so that it's easier to track. I would encourage you to do the same, unless already handled, so that this doesn't end up living there forever!

Reply:
As a heads up in case you missed the slack thread, this whole page redesign is in progress with unfinished UX being expected. There is a outlined implementation plan in the description.

Reply:
Sounds good. That means there's JIRA ticket in progress. I would myself use the umbrella ticket as a reference to quickly find all the remaining work under that ticket, but I guess you're deeper into the NFT part of the codebase, so do what works better for you!